nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -97,13 +107,23 @@
             .addValidator(StandardValidators.createRegexValidator(0, 1, false))
             .defaultValue(null)
             .build();
+    public static final PropertyDescriptor DICTIONARY_ENTRY_METADATA_DEMARCATOR = new PropertyDescriptor.Builder()
+            .name("Dictionary Entry Metadata Demarcator")
```

Comment:
The preferred code style is to use a "machine-readable" (aka no spaces) value for `name` and provide a human-formatted value for `displayName`.